config_script/whitelabel.py

```diff
@@ -561,18 +561,6 @@ def set_flags_from_mobile_config(self):
                 with open(self.config_project_path, 'r') as openfile:
                     project_file_string = openfile.read()
                 errors_texts = []
-                # iterate for all configurations
-                for name, config in configurations.items():
-                    if 'env_config' in config:
-                        # get folder name for mobile config for current configuration by env_config 
-                        config_folder = config_settings.get(self.CONFIG_MAPPINGS, {}).get(config['env_config'])
-                        if config_folder:
-                            # replace fullstory flag
-                            project_file_string = self.replace_fullstory_flag(project_file_string, config_directory, name, config_folder, errors_texts)
```

Comment:
@IvanStepanok Maybe it would be better to comment out that line, rather than removing the whole ‘for’ loop?
Without this part, the whole `set_flags_from_mobile_config(self):` function is meaningless. We could leave this as an example how to work with other flags in project file. WDYT? Thank you

Reply:
makes sense, thanks!